components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js

```diff
@@ -10,18 +11,37 @@ export function AccountVerificationFormStep3LoadingSteps() {
   // State for managing hiding/showing of the resume in background modal
   const [isResumeModalOpen, openResumeModal, closeResumeModal] = useTernaryState(false);
 
-  const { basiqConnection, goForward } = useAccountVerificationForm();
+  const { basiqConnection ,finish,updateAccountVerificationFormState} = useAccountVerificationForm();
   const { error, progress, completed, stepNameInProgress, reset, setJobId } = basiqConnection;
 
   useEffect(() => {
     const newJobId = new URLSearchParams(window.location.search).get("jobId");
     setJobId(newJobId);
-  }, [setJobId])
+  }, [])
 
+  const { data } = useAccountsData({
+    userId: sessionStorage.getItem("userId"),
+  });
+
+  const errorOrNoData = error || !data || data.length === 0;
+  async function autoSelectedAccount () {
```

Comment:
Why did you create a function to select an account automatically ? You can put this logic in fetchAccounts method and set currentAccount